extensions/amp-story/0.1/amp-story.js

```diff
@@ -627,6 +637,9 @@ export class AmpStory extends AMP.BaseElement {
         this.element.querySelector('amp-story-page'),
         'Story must have at least one page.');
 
+    const firstPageId = (this.win.history && this.win.history.state &&
```

Comment:
nit: maybe this should be called `initialPageId`?

Reply:
done